android/app/build.gradle

```diff
@@ -45,8 +45,8 @@ android {
 
     defaultConfig {
         applicationId appProperties['id']
-        minSdkVersion 21
-        targetSdkVersion 31
+        minSdkVersion 24
```

Comment:
Should this be reverted to 21?

Reply:
No, in order to set Extras to Clipboard descriptions it needs to be, at least, at API >= 24 (android N) :/